nyc_taxis/challenges/indexing-querying.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "indexing-querying",
+  "description": "Indexes half the data for NYC Taxis, then concurrently queries Elasticsearch while indexing the remaining data.",
+  "default": false,
+  "schedule": [
+    {
+      "operation": "delete-index"
+    },
+    {
+      "operation": {
+        "operation-type": "create-index",
+        "settings": {%- if index_settings is defined %} {{index_settings | tojson}} {%- else %} {
+        "index.codec": "best_compression",
+        "index.refresh_interval": "30s",
+        "index.translog.flush_threshold_size": "4g"
+      }{%- endif %}
+    }
+  },
```

Comment:
formatting nit: several of these, esp. when there's jinja2 involved are not aligned.

I cleaned them up, here's the diff:

```
$ git diff
diff --git a/nyc_taxis/challenges/indexing-querying.json b/nyc_taxis/challenges/indexing-querying.json
index 6f500b0..9855d09 100644
--- a/nyc_taxis/challenges/indexing-querying.json
+++ b/nyc_taxis/challenges/indexing-querying.json
@@ -13,9 +13,9 @@
         "index.codec": "best_compression",
         "index.refresh_interval": "30s",
         "index.translog.flush_threshold_size": "4g"
-      }{%- endif %}
-    }
-  },
+        }{%- endif %}
+      }
+    },
     {
       "name": "check-cluster-health",
       "operation": {
@@ -45,10 +45,10 @@
       "operation": {
         "operation-type": "force-merge",
         "request-timeout": 7200{%- if force_merge_max_num_segments is defined %},
-    "max-num-segments": {{ force_merge_max_num_segments | tojson }}
-    {%- endif %}
-  }
-  },
+        "max-num-segments": {{ force_merge_max_num_segments | tojson }}
+        {%- endif %}
+      }
+    },
     {
       "name": "refresh-after-force-merge",
       "operation": "refresh"
@@ -128,4 +128,4 @@
       }
     }
   ]
-}
\ No newline at end of file
+}
```

Reply:
addressed, and took out the conditional comma in the force-merge part for readability

Reply:
Thanks, looks good